admin_console/views/scripts/partner/kmc-users.phtml

```diff
@@ -36,7 +36,6 @@
 				<td>
 					<select class="options" onchange="doAction(this.value, '<?php echo $user->partnerId; ?>','<?php echo $user->id; ?>')">
 						<option value=""><?php echo $this->translate('Select Action'); ?></option>
-						<option value="kmcLogin"><?php echo $this->translate('Login'); ?></option>
 						<option value="kmcNewLogin"><?php echo $this->translate('Login-new'); ?></option>
```

Comment:
if we are removing it lets also align with the publisher screen and change Login-new to Login KMCNG or something similar

Reply:
Using https://github.com/kaltura/server/commit/8556533878fdf5c1949426f8a479f696b2d346da#diff-4af3370c924849849caf2a5ba62465520a07a4032fa2fce4b77e43b47ad30fd4L87 as standard, fixed now.